pkg/storegateway/series_chunks_test.go

```diff
@@ -574,6 +589,40 @@ func TestLoadingSeriesChunksSetIterator(t *testing.T) {
 			loadErr:      errors.New("test err"),
 			expectedErr:  "test err",
 		},
+		"loads single set from single block with some ignored histogram chunks": {
+			existingBlocks: []testBlock{block3},
+			setsToLoad: []seriesChunkRefsSet{
+				{series: []seriesChunkRefs{toSeriesChunkRefs(block3, 0), toSeriesChunkRefs(block3, 1)}},
+			},
+			ignoreNativeHistograms: true,
+			expectedSets: []seriesChunksSet{
+				{
+					series: func() []seriesEntry {
+						seriesEntries := make([]seriesEntry, 0, len(block3.series))
+
+						for i := range block3.series[0:2] {
+							refs := make([]chunks.ChunkRef, 0, len(block3.series[i].chks))
+							chks := make([]storepb.AggrChunk, 0, len(block3.series[i].chks))
+
+							for j := range block3.series[i].chks {
+								if j%2 != 0 {
```

Comment:
nit: Do you want to check for the nil `Raw` instead?

Reply:
Oop yea, I think I do--that way if the way chunks are chosen to be native histogram changes, it won't break here at least.